descriptors/ModuleDescriptor-template.json

```diff
@@ -15,11 +15,22 @@
           "methods": [ "POST" ],
           "pathPattern": "/refreshtoken",
           "permissionsRequired": [ "auth.signrefreshtoken" ]
+        }
+      ]
+    },
+    {
+      "id": "authtoken2",
```

Comment:
Given that permission `auth.signtoken` is now shared between two interfaces I would have preferred to simply bump authtoken to 2.1.

Reply:
We went with the 2 groups so that we have some way to represent on the top level how there are now two different interfaces.